tests/reducer-tests.js

```diff
@@ -0,0 +1,47 @@
+import reducer from '../app/reducer/reducer';
+
+module('reducer', () => {
+  test('it exists', (assert) => {
+    assert.ok(reducer, 'reducer exists');
+  });
+
+  test('character: CHARACTERS@FIND_ALL_COMPLETE', (assert) => {
+    const startingState = {};
+    const action = { type: 'CHARACTERS@FIND_ALL_COMPLETE', data: { name: 'Pad Thai', superpower: 'fluffiness' } };
+    const expectedState = { name: 'Pad Thai', superpower: 'fluffiness' };
+
+    assert.deepEqual(reducer(startingState, action).characters, expectedState, 'Character data is added');
+  });
```

Comment:
For each of your reducer tests, since you are tested a combined reducer, you will need to include all of the parts of your reducer.

So here is an example test:

```test('character: CHARACTERS@FIND_ALL_COMPLETE', (assert) => {
    const startingState = { characters: [], comics: [], series: null, modal: null };
    const action = { type: 'CHARACTERS@FIND_ALL_COMPLETE', data: [{ name: 'Pad Thai', superpower: 'fluffiness' }] };
    const expectedState = { characters: [{ name: 'Pad Thai', superpower: 'fluffiness' }], comics: [], series: null, modal: null };

    assert.deepEqual(reducer(startingState, action), expectedState, 'Character data is added');
  });```